sql/lesson2/tutorial.md

```diff
@@ -0,0 +1,616 @@
+
+```SQL
+SELECT name
+  FROM people
+ WHERE age = 18
+    OR age = 21
+    OR age = 30;
+```
+
+This query works, but you can write it in a more readable way, thanks to the operator **IN**
+```SQL
+SELECT name
+  FROM people
+ WHERE age IN (18,21,30);
+```
+
+Try these two in the database. Do you get the same results? The **IN** operator must be followed by a list of values. All the records having any of the values in the list for the given column (age in this example) will be kept in the result.
+
+You can also negate the operator IN, using **NOT IN**. The results is pretty straight forward. Contrary to the IN alone, when using NOT IN, only the record for which their value is NOT in the list will be kept in the result.
+
+```SQL
+SELECT name
+  FROM people
+ WHERE age NOT IN (18,21,30);
+```
+
+Try it in the Database. What do you get now?
+
+#### LIKE operator
+
+Another operator you can use to select multiple records at a time is the operator **LIKE**. This operator is used to filter text columns with partial matches.
+
+When using **LIKE**, there are two wildcards you can use in your conditions. Those are:
+
+* % - The percent sign represents zero, one, or multiple characters
+* _ - The underscore represents a single character
+
+For example:
+```SQL
+SELECT name
+  FROM people
+ WHERE name LIKE '%li%';
+```
+
+Will return Olivia Juanes and William Brown. Both have _li_ in them.
+
+The following one, will return those starting with _M_:
+```SQL
+SELECT *
+  FROM people
+ WHERE name LIKE 'M%';
+```
+
+And this one, those ending with _s_:
+```SQL
+SELECT *
+  FROM people
+ WHERE name LIKE '%s';
+```
+
+If using the _ then this one will return any name where there is an _m_, followed by any character, and then followed by an _r_:
+```SQL
+SELECT *
+  FROM people
+ WHERE name LIKE '%m_r%';
+```
+
+### The NULL value
+
+As defined in [Wikipedia](https://en.wikipedia.org/wiki/Null), **NULL** is a special marker used in SQL to indicate that a data value does not exist in the database. Empty data values lead to a whole subject (_Three valued logic_) that we'll cover in future lessons, but if you are curious you can read more  [here](https://en.wikipedia.org/wiki/Three-valued_logic#Application_in_SQL). For now, we'll just use **IS** and **IS NOT** operators to filter records containing NULL. Two examples below:
+
+```SQL
+SELECT *
+  FROM people
+ WHERE studies IS NULL;
+```
+
+```SQL
+SELECT *
+  FROM people
+ WHERE studies IS NOT NULL;
+```
+
+Study the results of each one. Easy huh?
+
+It's time to do some exercises now... :)
+
+## Lesson 2 exercises
+
+For the exercises below, we'll use the table from previous lesson: **people** and we'll introduce four new ones: **locations**, **hosts**, **workshops** and **attendance**. The next sections show a few rows as examples of the values. If you want to see the whole table, you'll find the fiddle link and the SELECT statement under each table.
```

Comment:
A few corrections, if I may:
`For the exercises below, we'll use the **people** table from the previous lesson, and we'll introduce four new tables: **locations**, **hosts**, **workshops** and **attendance**.`
I'm not sure about the value of listing the fields and sample data - we haven't taught them table defs or insert statements, so we should probably just be pointing them straight to the fiddle hashes at the bottom of the tutorial?

Reply:
I think a visual representation of the tables can help the coaches and the students in the learning process. I suggest leaving them there and if we find them confusing we can remove them in a next iteration.

The first sentence was rephrased as suggested.